pkgs/applications/misc/nb/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, lib, fetchFromGitHub, installShellFiles
+, bash, git
+}:
+stdenv.mkDerivation rec {
+  version = "5.7.8";
```

Comment:
```suggestion
  version = "6.10.1";
```

Reply:
I don't currently have time to. The script is enormous and somewhat messy and this interacts poorly with wrapping.